docs/_static/rtd_dark.css

```diff
@@ -11,6 +11,18 @@
  * Modified by Nathan Dyer: 20230103
  */
 
+:root {
```

Comment:
@tina-ux note the color definitions here - I've specified a bunch of greys just to get the functionality in place but am happy to modify for specific values.